plugins/node/opentelemetry-instrumentation-knex/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@opentelemetry/instrumentation-knex",
+  "version": "0.16.0",
+  "description": "OpenTelemetry Knex automatic instrumentation package",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.ts'",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "tdd": "yarn test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p .",
+    "prepare": "npm run compile",
+    "watch": "tsc -w"
+  },
+  "keywords": [
+    "opentelemetry",
+    "knex",
+    "nodejs",
+    "tracing",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.5.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@opentelemetry/context-async-hooks": "0.19.0",
+    "@opentelemetry/node": "0.19.0",
+    "@opentelemetry/tracing": "0.19.0",
+    "@types/mocha": "7.0.2",
+    "@types/node": "14.0.27",
+    "codecov": "3.7.2",
+    "gts": "3.1.0",
+    "knex": "^0.95.6",
+    "mocha": "7.2.0",
+    "nyc": "15.1.0",
+    "rimraf": "3.0.2",
+    "sqlite3": "^5.0.2",
+    "ts-mocha": "8.0.0",
+    "typescript": "4.1.3"
+  },
+  "dependencies": {
+    "@opentelemetry/api": "^1.0.0-rc.0",
+    "@opentelemetry/instrumentation": "^0.19.0",
```

Comment:
instrumentations in master already bumped version to `0.20.0`.
Probably best to also update here before merging (and also bump the version field to be `0.20.0`)